### PR TITLE
[bazel] Automate download of `bazelisk`

### PR DESCRIPTION
**Pull request by cfrantz**
1. Create downloader script.
2. Git ignore //.bin since we dont want to commit binaries to the codebase.
Signed-off-by: Chris Frantz <cfrantz@google.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -73,3 +73,6 @@ sw/host/**/target
 
 # Bazel-related cache and output directories
 bazel-*
+
+# Local binaries that should not be committed to the codebase
+third_party/bin
```
Comment (mcy) [inline .gitignore:78]: Could this be a dot-file at the top of the repo, instead...?

File: util/bazelisk.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```
Comment (mcy) [inline util/bazelisk.sh:1]: Arguably this file is so important it should live at the top of the repo, so people write `./bazelisk.sh`.

I am low-key a fan of the name of Rust's cursed make/cargo wrapper, `x.py`, so maybe we should go to the extreme and name it `./bz.sh`.

Comment (mcy) [inline util/bazelisk.sh:1]: Also, not to ask the obvious, but why shell instead of python? To make it easier to exec into bazelisk?

**Review by mcy — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (cfrantz) [reply .gitignore:78]: Done

File: util/bazelisk.sh (hunk 2 above)
Comment (cfrantz) [reply util/bazelisk.sh]: Named `bazelisk.sh` in t he root of the project.

Comment (cfrantz) [reply util/bazelisk.sh]: Lowest common denominator.

**Review by drewmacrae — APPROVED** [review]

**Pull request merged**